pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -2,6 +2,7 @@ const { I, adminPage } = inject();
 const assert = require('assert');
 
 const locateLabel = (dataQA) => locate(`[data-qa="${dataQA}"]`).find('span');
+const communicationDocsLink = 'https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/configure.html#communication';
```

Comment:
nit: I would suggest we put all URLs into one object (one place) and reference it wherever needed. So much easier to reason about them all :)

Reply:
Right now they are spread all over the code.

Reply:
makes sense, will do, thanks